wandb/sdk/wandb_run.py

```diff
@@ -2134,6 +2134,27 @@ def _swap_artifact_name(self, artifact_name: str, use_as: Optional[str]) -> str:
     def _detach(self) -> None:
         pass
 
+    def link_artifact(
+        self,
+        artifact: Union[public.Artifact, Artifact],
+        portfolio_path: str,
+        aliases: List[str],
+    ) -> None:
+        portfolio, project, entity = wandb.util._parse_entity_project_item(
```

Comment:
Same as other one - probably want to use same logic as artifacts

Reply:
Same response -- I'm not a fan of the artifact parse func because it pulls entity and project from the settings. That logic should be external to the function.